src/encoded/types/access_key.py

```diff
@@ -79,6 +82,14 @@ class Collection(Item.Collection):
 @view_config(context=AccessKey.Collection, permission='add', request_method='POST',
              validators=[validate_item_content_post])
 def access_key_add(context, request):
+    principals = effective_principals(request)
+    has_permission = any([
+        principal
+        for principal in principals
+        if principal in ('group.admin') or principal.split('.')[0] in ('submits_for', 'viewing_group',)
```

Comment:
This does not seem right. But I could not think of a better way.

Reply:
Yeah this isn't how ACLs work. I think the correct approach is changing who is assigned add permission here: https://github.com/ENCODE-DCC/encoded/blob/fa8ff2d166b89bbce4933005adca2bbf5f552702/src/encoded/types/access_key.py#L41 Currently it is `Authenticated` which is a Pyramid catchall for anyone who is logged in.